Source/debug.cpp

```diff
@@ -238,6 +238,7 @@ std::string DebugCmdLoadQuestMap(const string_view parameter)
 		if (!MyPlayer->isOnLevel(quest._qlevel)) {
 			StartNewLvl(*MyPlayer, (quest._qlevel != 21) ? interface_mode::WM_DIABNEXTLVL : interface_mode::WM_DIABTOWNWARP, quest._qlevel);
 			ProcessMessages();
+			ProcessMessages();

```

Comment:
I know it's only a debug command but perhaps we should add a explaining topic for this workaround?

```suggestion
			// Workaround for SDL_PollEvent:
			// StartNewLvl pushes a new event with SDL_PushEvent.
			// ProcessMessages calls SDL_PollEvent but SDL ignores the new pushed event.
			// Calling SDL_PollEvent again fixes this.
			ProcessMessages();
```